app.js

```diff
@@ -3,19 +3,31 @@ import express from 'express'
 import nodeCleanup from 'node-cleanup'
 import routes from './routes.js'
 import { init, cleanup } from './whatsapp.js'
+var cors = require("cors")
 
 const app = express()
+app.use(cors())
+
 const host = process.env.HOST ?? '127.0.0.1'
+const onHeroku = process.env.HEROKU ?? false
```

Comment:
Will Heroku always set this env var?

I think it's better to just check for the `HOST` env var instead, so that it will also support other platforms too.

Reply:
No heroku will not set it the person deploying to heroku will set it to be true in env so that it will work in heroku ,if it will not be set i will work like before

Reply:
I see, i'll try this approach after this merge:

> I think it's better to just check for the `HOST` env var instead, so that it will also support other platforms too.